all_test.go

```diff
@@ -23,6 +23,10 @@ TODO: Add test that verifies that SetPassword does not create an identity if tha
 Create a test Postgres database:
 	create database authaus_test owner = authaus_test;
 
+Create a test for password expiry (PasswordExpiresAfter).
+
+Create a test for users exempt from expiry (UsersExemptFromExpiring).
```

Comment:
So `Create a test Postgres database` is an instruction :P not a TODO. Please move these two comments to below the TODO, and give them each their own TODO, and reword to `Add test` instead of `Create a test`